### ISSUE TITLE
Source Code Would Make Our Lives Better

### ISSUE THREAD

**Issue by tiagosampaio** [mage2tv/issues#47]
@Vinai,

Is there any place where I can find the examples source code? I think it would save our lives a lot.

Thank.

**Comment by Vinai** [comment]
Hi @tiagosampaio, thanks for opening this issue.
Currently the source code from the examples is not published. Maintaining access credentials and source code would take away time from producing more content, so I decided against it (for now).

Is there a specific piece of code you are interested in?

**Comment by bayer** [comment]
Hi Vinai, 
I would be interrested in the code used in the hole punching / custom data-sections videos, since I'm having trouble following them - is it possible to obtain the code?

**Comment by Januszpl** [comment]
Yes. Source code would be helpful. Easier to check something than looking in video to find specific second to see how code look to check/verify things with our own code.

Currently files from GraphQL:
Creating a simple RequireJS based JavaScript test runner (12 minutes)seen
Calculating the currentPage and pageSize for GraphQL queries (21 minutes)seen

Would be useful to me.

**Comment by Vinai** [comment]
@bayer and @Januszpl Please send me an email to contact@mage2.tv and I'll make those code archives available to you.
